magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -207,12 +184,34 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
 
     zenith = 90 - sim_config["max_alt"].to_value("deg")
     azimuth = Angle(sim_config["max_az"]).wrap_at("360 deg").degree
-
+    logger.info(np.asarray(list(assigned_tel_ids.values())))
+    LSTs_IDs = np.asarray(list(assigned_tel_ids.values())[0:4])
```

Comment:
this would fail if LSTs are not put as the first ones in the config file, or if e.g. LST2-4 are removed from them.
why not simply loop over keys and pick up those starting with LST?

Reply:
Thanks for pointing this out. In the past few days, I've been discussing with Elisa and Federico about this issue. The solution of simply looping over keys starting with LST isn't the best because it would allow for typos like "LSTT-1". We then decided to create a new function in io.py called "check_input_list" that checks if the input telescope list is in agreement with the following rules:
 1) All 4 LSTs and 2 MAGICs must be listed
 2) All 4 LSTs must come before the MAGICs

Besides that, the input list is quite flexible. For instance, one can write a list as:
mc_tel_ids:
        LST-4: 1
        LST-2: 7
        LST-3: 9
        LST-1: 0
        MAGIC-II: 2
        MAGIC-I: 3

and it will work fine.
If the list does not follow these rules, the function check_input_list() will raise an exception and point exactly what was the error to the user.

Reply:
great, thanks for implementing this!